apps/condo/domains/billing/schema/BillingIntegration.test.js

```diff
@@ -48,6 +49,33 @@ describe('BillingIntegration', () => {
                 expect(err).toBeDefined()
             })
         })
+
+        test('Can be created with options', async () => {
+            const support = await makeClientWithSupportUser()
+            const firstOption = { name: '1C', billingPageTitle: 'Биллиг "Реестры 1C"', descriptionDetails: { detailsText: 'о формате', detailsLink: faker.internet.url() } }
+            const secondOption = { name: 'Сббол 9_2', billingPageTitle: 'Биллиг "Реестрыыыыыы"' }
+            const thirdOption = { name: 'Сббол 8_1', descriptionDetails: { detailsText: 'о формате', detailsLink: faker.internet.url() } }
```

Comment:
I need some documentation on how to create billing with options

Reply:
I'll write confluence page if needed, but for now example can be found in `bin/generate-billings`